src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/settings.html.twig

```diff
@@ -139,27 +143,38 @@
 
 <script>
 (function() {
-    let configOptions = document.querySelectorAll('.config-option');
-    let oppositeValues = { 'light': 'dark', 'dark': 'light', 'normal': 'full', 'full': 'normal' };
+    const configOptions = document.querySelectorAll('.config-option');
+    const allSettingValues = ['theme-auto', 'theme-ligh', 'theme-dark', 'width-normal', 'width-full'];
     [...configOptions].forEach(option => {
         option.addEventListener('click', function (event) {
```

Comment:
This should probably be implemented using the `change` event instead of the `click` event on the option. this way, it's more compatible with other ways to interact with the form.